src/eda.R

```diff
@@ -114,6 +109,21 @@ birthday_before_date <- function(birth_date, other_date) {
   }
 }
 
+
+imputed_birthday <- function(birth_year_month, report_date) {
```

Comment:
This function is already updated in master. The implementation there also takes account of the episodes data to protect against children being negative ages at join or over 18 when they leave, so I think it's preferable.

Reply:
I thought I had covered that, but I'll take your word for it and switch it back to your implementation 👍